main/guides/agoric-cli/README.md

```diff
@@ -1,59 +1,96 @@
-# Agoric CLI Commands
+# Agoric CLI Reference
 
-In general, you'll want to issue the Agoric CLI (Command Line Interface) commands in this order:
+## agoric help
+
+Displays the Agoric CLI commands and arguments with brief descriptions.
+
+**Usage**:
+
+```sh
+agoric [command]
+```
+
+By default, lists all commands:
+
+```
+  cosmos <command...>                            client for an Agoric Cosmos chain
+  ibc-setup <command...>                         set up Inter Blockchain Communication
+  open [options]                                 launch the Agoric UI
+  init [options] <project>                       create a new Dapp directory named <project>
+  set-defaults [options] <program> <config-dir>  update the configuration files for <program> in <config-dir>
+  ibc-relayer                                    run an Inter Blockchain Communications relayer
+  install [force-sdk-version]                    install Dapp dependencies
+  follow [options] <path-spec...>                follow an Agoric Casting leader
+  run <script> [script-args...]                  run a script with all your user privileges and some Agoric endowments
+  deploy [options] [script...]                   run multiple scripts with all your user privileges against the local Agoric
+                                                 VM
+  publish [options] [bundle...]                  publish a bundle to a Cosmos chain
+  wallet [options]                               wallet commands
+  start [options] [profile] [args...]            start an Agoric VM
+  help [command]                                 display help for command
+```
+
+## agoric run
+
+Run a script with all your user privileges and some Agoric endowments.
+
+**Usage**:
 
-1. `agoric init $projectDir`
-2. `cd $projectDir`
-3. `agoric install`
-4. `agoric start` (Usually with `--reset`)
-5. `agoric deploy`
-6. `agoric open`
+```sh
+agoric run [options] <script> [script-args...]
+```
+
+See also:
 
+- [`agoric run` supports bundling proposal modules with writeCoreProposal](https://github.com/Agoric/agoric-sdk/discussions/8087#discussioncomment-6534378) discussion [#8087](https://github.com/Agoric/agoric-sdk/discussions/8087) 2023-07
 
-## agoric init
+## ~~agoric init~~
+
+_Deprecated in favor of [yarn create @agoric/dapp](../getting-started/#creating-your-dapp-from-a-template)_
```

Comment:
I don't know the right verbiage, but it may be worthwhile to say something like:
```suggestion
_Internal implementation for the preferred [yarn create @agoric/dapp](../getting-started/#creating-your-dapp-from-a-template)_
```
if we removed `agoric init`, then the `yarn create @agoric/dapp` form would stop working.